api/sheets/index.md

```diff
@@ -8,38 +8,58 @@ has_children: true
-users to import data from the Data Commons knowledge graph. The add-on provides an
-interface for finding a location's [DCID](/glossary.html), as well as some custom functions for
-importing data to a spreadsheet.
+The **Data Commons Sheets API** is a Google Sheets add-on that enables Google Sheets users to import data from the Data Commons knowledge graph. The add-on provides an interface for finding a location’s unique Data Commons identifier ([DCID](glossary.html)), and some custom functions for importing data into a spreadsheet.
+
+> **NOTE:**
+> 
+> For more information on DCIDs, refer to [Unique Identifiers (dcid)](/bigquery/unique_identifiers.html).
 
 Also read our [step-by-step guides](tutorials/) for more examples.
 
-## Getting started
+## Installing and Enabling the Sheets Add-On
+
+Install and enable the Sheets add-on from Google Workspace Marketplace, as follows:
 
-Install the add-on from [G Suite Marketplace](https://gsuite.google.com/marketplace/app/data_commons/454343067575). To enable the add-on in a document, click "Add-ons > Data Commons > Fill place dcids". You may use the resulting sidebar to start [finding dcids](/api/sheets/get_dcid.html) in the United States, or close it and reopen it at any time. Note that none of the custom functions will work in a given document until you have enabled the add-on by choosing "Fill place dcids".
+1. Install the Sheets add-on by clicking **[Google Workspace Marketplace](https://gsuite.google.com/marketplace/app/data_commons/454343067575)** .
+2. Enable the Sheets add-on by clicking the Google Sheets **Extensions** menu, pointing to **Data Commons**, and then by clicking **Fill place dcids**.
+3. Use the resulting sidebar that appears on the right side of the page to start [finding DCIDs](/api/sheets/get_name.html) in the United States. You can close and reopen the sidebar at any time.
 
 ![](/assets/images/sheets/sheets_menu_bar.png)
 
-## Finding a place's DCID
+> **NOTE:**
+> 
+> None of the custom Data Commons Sheets functions (described below) will work in a given document until you have enabled the add-on by opening the **Fill place dcids** sidebar.
 
-Data Commons' Sheets add-on ships with the ability to look up a location's [DCID](/glossary.html), or unique Data Commons identifier, within the application. To use this feature, select the final destination cell of the place [DCID](/glossary.html). Go to the Sheets menu bar, and click:
+## Finding a Place’s DCID
 
-**"Add-ons > Data Commons > Fill place dcids"**
+Data Commons’ Sheets add-on provides the ability to look up a location’s [DCID](/glossary.html), within the application by using the **Fill place dcids** feature. To use this feature to find a place’s DCID:
 
-![](/assets/images/sheets/sheets_menu_bar.png)
+1. Select the destination cell where you want to add a place’s [DCID](/glossary.html).
+2. Click in the search box for the **Fill place dcids for selected cells** section.
```

Comment:
Can we be more explicit about where this box is, like in the former text?

Reply:
I added some additions to the text. Please let me know what you think.